src/components/containers/SubplotAccordion.js

```diff
@@ -111,23 +118,34 @@ class SubplotAccordion extends Component {
             canDelete={false}
             messageIfEmpty={messageIfEmptyFold}
             subplot={layoutKey}
-            name={layoutKey}
+            name={subplotName}
           >
             {children}
           </NonCartesianSubplotFold>
         );
       }
     });
 
+    let pieCounter = 0;
```

Comment:
hehe OK, we'll refactor that next time, when we add a third :)

Reply:
that's what I was thinking when I was writing it lol